src/pages/App.tsx

```diff
@@ -252,7 +252,14 @@ export default function App() {
 
                   {nftFlag === NftVariant.Enabled && (
                     <>
-                      <Route path="/nfts" element={<NftExplore />} />
+                      <Route
+                        path="/nfts"
+                        element={
+                          <Suspense fallback={null}>
```

Comment:
I blue circle shows up by default and this removes that blue circle

Reply:
this isn't the correct solution, we should create a suspense fallback for the explore page but this would just be replaced by the Apollo migration and would be throwaway work. Can you add the comment : `// TODO: replace loading state during Apollo migration` so that is tracked